readme.md

```diff
@@ -99,6 +99,10 @@ install target/release/choose <DESTDIR>
 
 Just make sure DESTDIR is in your path.
 
+### Packaging status
+
+Fedora/CentOS [COPR](https://copr.fedorainfracloud.org/coprs/atim/choose/): `sudo dnf copr enable atim/choose -y && sudo dnf install choose`
```

Comment:
1. Let's get rid of `sudo`. Leave it up to the user.
2. Let's get rid of `-y`. Leave it to the user to decide these things.
3. Let's put the code in a proper code block with three backticks.
4. Split both commands on separate lines.